### DISCUSSION TITLE
Версия 1.1.5 (preview)

### DISCUSSION THREAD

**Discussion by qzeleza** [qzeleza/kvas#75] [Announcements]
- В данной версии, для соединения ssr теперь не происходит переподключение интернета.

<hr /><em>This discussion was created from the release <a href='https://github.com/qzeleza/kvas/releases/tag/v1.1.5'>Версия 1.1.5 (preview)</a>.</em>

**Comment by Rbuha** [comment]
kvas debug
-----------------------------------------------------------------------------------
КВАС версии 1.1.5
-----------------------------------------------------------------------------------
Текущая дата и время Thu Oct 12 15:36:23 +07 2023

-----------------------------------------------------------------------------------
Информация о роутере
-----------------------------------------------------------------------------------
Страна: EA
Модель: KN-1811
Продукт: Ultra
Тип архитектуры: aarch64

Текущее VPN соединение: ultra1 (Wireguard0)                         ГОТОВО
AdGuard Home НАСТРОЕН, как основной DNS сервер
УСТАНОВЛЕН список разблокировки ПО УМОЛЧАНИЮ.
-----------------------------------------------------------------------------------
Установка завершена: 12/10/2023 14:52:42
-----------------------------------------------------------------------------------
Список ошибок из системного журнала
-----------------------------------------------------------------------------------
 Opkg::Manager: /opt/etc/ndm/netfilter.d/100-vpn-mark: exit code 2.
 Oct 12 15:35:41

Состояние службы dnsmasq по команде
/opt/etc/init.d/S56dnsmasq status                                   ОСТАНОВЛЕНА
-----------------------------------------------------------------------------------

dnsmasq: failed to create listening socket for port 53: Address already in use

[] Проблемы при выделении адресов для аренды DHCP

При установки выбран adguard, 1.1.3 работает нормально.
